src/tests/ftest/network/cart_self_test.py

```diff
@@ -46,6 +46,7 @@ def __init__(self, path=""):
 
             self.repetitions = FormattedParameter("--repetitions {0}")
             self.attach_info = FormattedParameter("--path {0}")
```

Comment:
I think this can be removed if it is daos agent aware

Reply:
My only concern here is if we dump attachinfo and then point selftest to it, how do we know it is daos agent aware?